.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.enabled": true,
+    "python.linting.flake8Enabled": true,
+}
```

Comment:
I committed this file because I found these settings useful. If we don't want them in the repo I can remove this commit and add the settings at the user (rather than workspace) level.